ned/ned.py

```diff
@@ -303,7 +302,7 @@ def dns_cmd_create(ctx):
 
 @click.command(name="list")
 @click.pass_context
-def dns_cmd_describe(ctx):
+def dns_cmd_list(ctx):
```

Comment:
The names of `dns_cmd_list` and `dns_cmd_describe` were switched

Reply:
good catch